### PR TITLE
accept more fields in create matter API

### PR DESCRIPTION
**Pull request by yneves**

### CODE REVIEW

File: src/db/_matters.js [hunk 1]
```diff
@@ -100,10 +100,15 @@ function * create(matter) {
...
     priorityDate: '0000-00-00'
   }, matter);
+
   const query = squel
     .insert()
     .into('matterdb');
-  fieldsToUpdate.forEach(f => query.set(f, m[f] === undefined ? '' : m[f]));
+  fieldsToUpdate.forEach(field => {
+    let value = record[field] === undefined || record[field] === '' ?
+      (/Date$/.test(field) ? '0000-00-00' : '') : record[field];
```
Comment (ntgn81) [inline src/db/_matters.js]: It's probably better to have a list of fields that are date fields than to rely on the field name

Comment (yneves) [inline src/db/_matters.js]: You're completely right, don't know what i was thinking lol.

**Comment by yneves** [comment]
@ntgn81 done

**Comment by ntgn81** [comment]
Great!

**Pull request merged**